Library/Formula/gitup.rb

```diff
@@ -0,0 +1,85 @@
+class Gitup < Formula
+  homepage "https://github.com/earwig/git-repo-updater"
+  url "https://github.com/earwig/git-repo-updater.git",
+    :revision => "51a482d7a25bed39ccc75780813c65bd666c1436",
+    :tag => "v0.2.3"
```

Comment:
Afraid not. We mandate the `tag` and `revision` now - The reasoning being that the tag can be retagged silently which has bitten us before. The `revision` also acts as a checksum in that regard.